tests/common/include/tests/common/material_interaction.inl

```diff
@@ -418,13 +422,12 @@ TEST(material_interaction, telescope_geometry_scattering_angle) {
     constexpr scalar q{-1.f};
     constexpr scalar iniP{10.f * unit<scalar>::GeV};
 
-    typename bound_track_parameters<transform3>::vector_type bound_vector;
-    getter::element(bound_vector, e_bound_loc0, 0) = 0.f;
-    getter::element(bound_vector, e_bound_loc1, 0) = 0.f;
-    getter::element(bound_vector, e_bound_phi, 0) = 0.f;
+    // Initial track parameters directing x-axis
+    typename bound_track_parameters<transform3>::vector_type bound_vector =
```

Comment:
So, you have one plane at zero and start at zero? Since this is not actually stepping anywhere, let's get rid of the step size constraints etc.

Reply:
OK